.github/workflows/test.yml

```diff
@@ -0,0 +1,50 @@
+name: Test
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  linux:
+    name: "Node ${{ matrix.node }} on Linux: Test and Lint"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
```

Comment:
All good now, just add a `fail-fast: false` in each `strategy` to match the previous behavior